<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain_zeitGypsum</class>
 <widget class="QMainWindow" name="frmMain_zeitGypsum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>793</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1091</width>
      <height>631</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 255, 234);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Production</string>
     </attribute>
     <widget class="QPushButton" name="btnProdTUpLoad">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>ProdT Uploade</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnServerLogin">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string>Login To Server</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>112</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 211, 211);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>112</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 220, 220);</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtPassword">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>g1234</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtID">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>leekr2</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbServerConnected">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Server Connected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnProdTRead">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>ProdT Data Read</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QDateTimeEdit" name="edtDayFrom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd </string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="edtDayTo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProduction">
    <property name="title">
     <string>Production</string>
    </property>
    <addaction name="action_ReadFromDB"/>
    <addaction name="action_DailyProd"/>
   </widget>
   <addaction name="menuProduction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_ReadFromDB">
   <property name="text">
    <string>Read From DB</string>
   </property>
  </action>
  <action name="action_DailyProd">
   <property name="text">
    <string>DailyProd</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnServerLogin</sender>
   <signal>clicked()</signal>
   <receiver>frmMain_zeitGypsum</receiver>
   <slot>ServerLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProdTUpLoad</sender>
   <signal>clicked()</signal>
   <receiver>frmMain_zeitGypsum</receiver>
   <slot>ProdTUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnProdTRead</sender>
   <signal>clicked()</signal>
   <receiver>frmMain_zeitGypsum</receiver>
   <slot>ProdTDataRead()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menubar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>frmMain_zeitGypsum</receiver>
   <slot>MenuAction(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>906</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>965</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MenuAction(QAction*)</slot>
  <slot>ServerLogin()</slot>
  <slot>ProdTUpdate()</slot>
  <slot>ProdTDataRead()</slot>
 </slots>
</ui>
